changelogs/fragments/resolve-issue-342.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - meraki_ms_switchport - resolve issue #342
```

Comment:
Please describe the change so when someone reads the changelog they know what was fixed. Although pointing to the issue number is helpful and something I should do more of.